algebird-core/src/main/scala/com/twitter/algebird/statistics/GaussianDistributionMonoid.scala

```diff
@@ -0,0 +1,45 @@
+package com.twitter.algebird.statistics
+
+import com.twitter.algebird.Monoid
+
+/**
+ * @mean - Mean
+ * @sigma - Standard deviation
```

Comment:
wait, this is not correct. It needs to be `sigma2` being the variance:

http://en.wikipedia.org/wiki/Sum_of_normally_distributed_random_variables

we can have a method to get the `stddev` to use the same name as: https://github.com/twitter/algebird/blob/develop/algebird-core/src/main/scala/com/twitter/algebird/MomentsGroup.scala
`def stddev = math.sqrt(sigma2)`